release_notes/370-release-notes.adoc

```diff
@@ -164,7 +164,7 @@ In the table, features are marked with the following statuses:
 * Fixed a permission issue for vulnerability reports where users with read/write permission could still not create or edit reports. (**ROX-9880**)
 * Fixed issue that caused connection problems to the {ocp} console after connecting to {product-title-short} or the inability to connect to {product-title-short} if a connection to the {ocp} console existed. Central will now respond with a `421 Misdirected Request` status code to requests where the `ServerName` sent via TLS SNI does not match the `:authority` (Host) header. This feature can be turned off by setting the environment variable `ROX_ALLOW_MISDIRECTED_REQUESTS=true`. (**ROX-9625**)
 * When editing a policy, the **Violations Preview** window was unavailable for disabled policies. This issue has been fixed. (**ROX-9435**)
-
+* Added the ability to disable role-based access control (RBAC) related risk computation. Users who do not want to include the RBAC factor in risk calculation can set the environment variable `ROX_INCLUDE_RBAC_IN_RISK=false` in the Central deployment spec. (**ROX-10627**)
```

Comment:
nit: Replace `the RBAC factor` with just `RBAC`

suggested for consideration: "Users can exclude RBAC from risk calculation by setting the environment variable ....."

Reply:
Thank you!